### ISSUE TITLE
.htaccess not up to date

### ISSUE THREAD

**Issue by kris711** [PrestaShop/PrestaShop#11098]
**Describe the bug**
/translation/cldr/ .htaccess is not updated as the others.
This  causes authentication errors in server logs which might be a problem if server is using fail2ban or sth similliar.

**To Reproduce**
1. Installation with Apache 2.4 
2. Edit a Product
3. Check Serverlog

**Serverlog**
AH01630: client denied by server configuration: /changedbecauseitsSecret/translations/cldr/datas/main/de-DE/numbers.json, referer: https://www.mydomain.com/admin/index.php/product/form/XXX?_token=changedbecauseitsSecret

AH01630: client denied by server configuration: /changedbecauseitsSecret/translations/cldr/datas/supplemental/currencyData.json, referer: https://www.mydomain.com/admin/index.php/product/form/XXX?_token=changedbecauseitsSecret

AH01630: client denied by server configuration: /changedbecauseitsSecret/translations/cldr/datas/supplemental/likelySubtags.json, referer: https://www.mydomain.com/admin/index.php/product/form/XXX?_token=changedbecauseitsSecret

AH01630: client denied by server configuration: /changedbecauseitsSecret/translations/cldr/datas/supplemental/plurals.json, referer: https://www.mydomain.com/admin/index.php/product/form/XXX?_token=changedbecauseitsSecret

AH01630: client denied by server configuration: /changedbecauseitsSecret/translations/cldr/datas/main/de-DE/currencies.json, referer: https://www.mydomain.com/admin/index.php/product/form/XXX?_token=changedbecauseitsSecret

**Additionnal information**
PrestaShop version: 1.7.4.2 and 1.7.4.3 (not earlier)
PHP version: 7.1.23

**Solution**
Change content in /translation/cldr/ .htaccess:

from:

```
Order deny,allow
Allow from all
```

to:

```
# Apache 2.2
<IfModule !mod_authz_core.c>
    Order deny,allow
    Allow from all
</IfModule>

# Apache 2.4
<IfModule mod_authz_core.c>
    Require all granted
</IfModule>
```

**Comment by MathiasReker** [comment]
It is already updated in 1.7.5.0

**Comment by khouloudbelguith** [comment]
Hi @kris711,

Here's the PR: https://github.com/PrestaShop/PrestaShop/pull/10347 to solve the issue.
I close the issue, feel free to open a new one when needed.
Thanks!

**Comment by khouloudbelguith** [comment]
Hi @kris711,

Here's the PR: https://github.com/PrestaShop/PrestaShop/pull/10347 to solve the issue.
I close the issue, feel free to open a new one when needed.
Thanks!

**Comment by khouloudbelguith** [comment]
Hi @kris711,

Here's the PR: https://github.com/PrestaShop/PrestaShop/pull/10347 to solve the issue.
I close the issue, feel free to open a new one when needed.
Thanks!

**Comment by khouloudbelguith** [comment]
Duplicate of #10347

**Comment by Peter4661** [comment]
We have this error on our live site, not on our dev site, which runs on the same server.

Both files /translation/cldr/ .htaccess: have the same content.
Order deny,allow
Allow from all

I changed content of .htaccess file on live site, did not solve the issue.

Server log:
[Tue Nov 20 09:08:15.496811 2018] [authz_core:error] [pid 1010862:tid 140538667788032] [client 217.62.77.185:50305] AH01630: client denied by server configuration: /home/xxxxxx/domains/www.mydomain/public_html/translations/cldr/datas/supplemental/plurals.json, referer: https://www.mydomain/xxx/index.php/product/form/6002?_token=Changedxxx

Server info:
Servergegevens Linux #1 SMP Wed May 2 07:43:19 EDT 2018 x86_64
Server softwareversie: Apache
PHP-versie: 7.2.10
Geheugenlimiet: 1024M
Maximale uitvoeringstijd: 30
Maximale bestandsgrootte voor uploads: 128M

**Comment by Peter4661** [comment]
Found this explanation: https://httpd.apache.org/docs/2.4/upgrading.html

I'm not expert in this field. Over to you.

**Comment by Peter4661** [comment]
Cont.

Changed back /translation/cldr/ .htaccess to:
Order deny,allow
Allow from all

No error in web log (anymore) regarding "client denied by server configuration" !
Error in front office is persistant.

Tested also with PHP-versie: 7.1.22, PHP-version: 7.0.32, PHP-version: 5.6.38,

**Comment by khouloudbelguith** [comment]
Hi @Peter4661,

This issue is fixed in the next release 1.7.5.0.
Here's the PR:  #10347 to solve the issue.
Thanks for your understanding!
